scripts/exrrfs_prep_lbc.sh

```diff
@@ -42,7 +42,7 @@ if [ -r "${checkfile}" ]; then
   for fhr in  ${fhr_all}; do
     CDATElbc=$(${NDATE} "${fhr}" "${CDATE}")
     string_time=$(date -d "${CDATElbc:0:8} ${CDATElbc:8:2}" +%Y-%m-%d_%H.%M.%S)
-    ${cpreq} "${COMINrrfs}/${RUN}.${YYYYMMDDlbc}/${HHlbc}/lbc/${WGF}${MEMDIR}/lbc.${string_time}.nc"  "${UMBRELLA_PREP_LBC_DATA}/."
+    ln -sf "${COMINrrfs}/${RUN}.${YYYYMMDDlbc}/${HHlbc}/lbc/${WGF}${MEMDIR}/lbc.${string_time}.nc"  "${UMBRELLA_PREP_LBC_DATA}/."
```

Comment:
@chunhuazhou Do not change line 45.

Two solutions:
1.  add an extra line before line 5:
`cpreq="ln -snf"`

The original idea is that in the development stage, we use links; but in operation, we use `cpreq` by defining it in the job cards.

2. More elegant solution is to add `<envar><name>cpreq</name><value>ln -snf</value></envar>` in the prep_lbc task in rrfs.xml. I am fine with either one at the moment.

Thanks!